data-tck/src/main/java/io/micronaut/data/tck/entities/Author.java

```diff
@@ -26,22 +26,17 @@
 import jakarta.persistence.OneToMany;
 
 @Entity
-@javax.persistence.Entity
```

Comment:
Both annotations were used when we supported both hibernate5 and hibernate6 for some period of time, javax not needed anymore and these generate compile warnings in hibernate modules so we can remove them now.